src/index.js

```diff
@@ -107,10 +107,12 @@ export async function run() {
       "tmate-server-ed25519-fingerprint": /./,
     }
 
-    for (const opt in options) {
-      const value = getValidatedInput(opt, options[opt]);
+    for (const [key, option] of Object.entries(options)) {
+      if (core.getInput(key) === '')
```

Comment:
```suggestion
      if (core.getInput(key) === '')
```
note: it always returns a string, so we should be good with checking for an empty string.